src/styles/index.scss

```diff
@@ -1,3 +1,21 @@
+// @import url(card.scss);
+// @import url(stars.scss);
```

Comment:
Use imports, to avoid incorrect styles at first import reset/normalize, variables, mixins, and then other files that use them